assembly/src/assembler/instruction/u32_ops.rs

```diff
@@ -481,29 +463,29 @@ fn calculate_clz(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyEr
     let ops_group_2 = [
         Push(Felt::new(u32::MAX as u64 + 1)), // [2^32, pow2(32 - clz), n, clz, ...]
 
-        Dup1, Neg, Add, // [2^32 - pow2(32 - clz), pow2(32 - clz), n, clz, ...] 
-                        // `2^32 - pow2(32 - clz)` is equal to `clz` leading ones and `32 - clz` 
```

Comment:
Yeah there were quite a few places, almost every module I touched honestly, that had some kind of formatting discrepancy, mostly really small things like this example, but having some kind of CI check would be helpful. I don't think pre-commit hooks are necessary as long as the PR status checks fail due to this stuff (and is more of a impetus for those that don't set up their editors to do this stuff for them, to do so and avoid the headache).

Reply:
> I don't think pre-commit hooks are necessary as long as the PR status checks fail due to this stuff

By `pre-commit` hook I mean the [tool](https://pre-commit.com/). I have added the [configuration](https://github.com/0xPolygonMiden/miden-vm/blob/main/.pre-commit-config.yaml) for it a while back. People don't have to install the tool if they don't want (and we dont' even have to use this tool to check the format in the CI), it is just an easy way of get this setup since the config is there already.

Reply:
I think we should add this to the README - I didn't know about that tool, or that we had that configuration already setup. I'll set it up now